README.md

```diff
@@ -1 +1,38 @@
 # eslint-config-inturn
+
+This is a config that contains the linting rules used by developers for Inturn!
+
+## Installation
+
+1. Install the config itself.
+```bash
+# install the config itself
+npm install --save-dev eslint-config-inturn
+```
+
+2. If you haven't already, install `install-peerdeps` globally.
+```bash
+npm install -g install-peerdeps
+```
+
+3. Install the peer dependencies of `eslint-config-inturn`.
+```bash
+install-peerdeps --dev eslint-config-inturn
+```
```

Comment:
`npx install-peerdeps --dev eslint-config-inturn` instead of installing it.

Reply:
Personally, I don't think it's necessary to install `install-peerdeps`, I would suggest just a gentle reminder to not forget to install peer deps and let the user decide how they would like to install them. Not a big deal either way.